### PR TITLE
changed method in animation tutorial table of methods

### PR DESCRIPTION
**Pull request by story645**
The animation tutorial has a table of plotting methods and the underlying artists to illustrate what to look for in the docs. I didn't catch on review that this has patches.Circle as the first entry. 

image

Since Ellipse isn't the underlying patch I change the first entry to `Axes.add_patch` b/c I like 'em all as axes methods, but I'm unsure of @chahak13 intent here. (My guess is the example started as circle and morphed to ellipse)

### CODE REVIEW

**Comment by chahak13** [comment]
Thanks! It did indeed start as a circle but I changed it to Ellipse since all the base methods came from Ellipse.

**Comment by chahak13** [comment]
I think I'd prefer the second one so that it's clear.

File: tutorials/introductory/animation_tutorial.py [hunk 1]
```diff
@@ -62,28 +62,28 @@
...
+#                                                                          PathCollection.set_offsets`
+# `.Axes.imshow`                            `.image.AxesImage`             ``AxesImage.set_data``
+# `.Axes.annotate`                          `.text.Annotation`             `.text.Annotation.\
+#                                                                          update_positions`
+# `.Axes.barh`                              `.patches.Rectangle`           `.Rectangle.set_angle`,
+#                                                                          `.Rectangle.set_bounds`,
+#                                                                          `.Rectangle.set_height`,
+#                                                                          `.Rectangle.set_width`,
+#                                                                          `.Rectangle.set_x`,
+#                                                                          `.Rectangle.set_y`
```
Comment (oscargus) [inline tutorials/introductory/animation_tutorial.py:79]:
```suggestion
#                                                                          `.Rectangle.set_y`,
```

**Comment by oscargus** [comment]
I'm not saying that I fully understand the idea here (either before or after this PR), especially if there are commas left out on purpose or just missing (the one I marked is an old one, but there I am quite sure it is simply missing).

It may be worth noting that I have not been able to get both `add_patch`and `Ellipse` to render on the same row.

Shouldn't `.Ellipse.set_center` et al end up in the last column?

File: tutorials/introductory/animation_tutorial.py [hunk 2]
```diff
@@ -62,28 +62,28 @@
...
+#                                                                          `.Rectangle.set_xy`
+# `.Axes.fill`                              `.patches.Polygon`             `.Polygon.set_xy`
+# `.Axes.add_patch` `.patches.Ellipse`      `.patches.Ellipse`             `.Ellipse.set_angle`,
+#                                           `.Ellipse.set_center`,
+#                                           `.Ellipse.set_height`,
+#                                           `.Ellipse.set_width`
```
Comment (chahak13) [inline tutorials/introductory/animation_tutorial.py:85]: Should be in the 3rd column, no?

**Review by QuLogic — APPROVED** [review]

**Pull request merged**